test/jdk/java/foreign/NativeTestHelper.java

```diff
@@ -117,7 +124,7 @@ public static boolean isPointer(MemoryLayout layout) {
     /**
      * The layout for the {@code double} C type
      */
-    public static final ValueLayout.OfDouble C_DOUBLE = ValueLayout.JAVA_DOUBLE;
+    public static final ValueLayout.OfDouble C_DOUBLE = ValueLayout.JAVA_DOUBLE.withBitAlignment(isAixOS ? 32 : ValueLayout.JAVA_DOUBLE);
```

Comment:
`withBitAlignment()` no longer exists; this should be
```java
    public static final ValueLayout.OfDouble C_DOUBLE = ValueLayout.JAVA_DOUBLE.withByteAlignment(isAixOS ? 4 : Double.BYTES);
```

Reply:
Fixed